client/my-sites/plans/jetpack-plans/product-store/hooks/use-get-original-price.tsx

```diff
@@ -21,8 +21,9 @@ export const useGetOriginalPrice = ( siteId: number | null ) => {
 				return product.displayPrice || -1;
 			}
 
-			const productSlug = product?.costProductSlug || product?.productSlug;
+			const productSlug = product.costProductSlug || product.productSlug;
```

Comment:
Drive-by change where `product` is always safely a valid object. The Jetpack CRM check few lines above would already fail if it wasn't.